src/layer/marker/Marker.Drag.js

```diff
@@ -58,10 +58,11 @@ L.Handler.MarkerDrag = L.Handler.extend({
 		}
 
 		marker._latlng = latlng;
+		e.latlng = latlng;
 
 		marker
-		    .fire('move', {latlng: latlng})
-		    .fire('drag');
+		    .fire('move', e)
```

Comment:
I don't like modified DOM event being passed as event data... In a similar situation in other place of Leaflet code, original event is passed as `e.originalEvent`.